### PR TITLE
Ansible format improvements

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

File: ansible/.yamllint [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+extends: default
+
+ignore: |
+  node_modules
```
Comment (brianhelba) [inline ansible/.yamllint]: Need to update this.

File: ansible/roles/minio/tasks/main.yml [hunk 2]
```diff
@@ -32,10 +32,20 @@
     timeout: 20
 
 - name: Create default bucket
-  shell: |
-    . {{ python_dist_path }}/bin/activate
-    export AWS_ACCESS_KEY_ID=accesskey
-    export AWS_SECRET_ACCESS_KEY=secretkey
-    {{ python_dist_path }}/bin/aws --endpoint-url http://localhost:9000 s3api create-bucket --bucket {{ isic_upload_bucket_name }} && touch {{ ansible_user_dir }}/.bucket.created
+  command: >-
```
Comment (brianhelba) [inline ansible/roles/minio/tasks/main.yml]: @danlamanna I refactored this command more heavily. Can you review.

Comment (danlamanna) [reply ansible/roles/minio/tasks/main.yml]: It turns out minio treats all directories within the top level datadir as buckets. So this while thing can just be simplified to creating `isic_upload_bucket_name`.

Comment (brianhelba) [reply ansible/roles/minio/tasks/main.yml]: So something like `creates: {{ isic_upload_bucket_name }}`? We should make this a fully qualified path, so where do the buckets live?

Comment (danlamanna) [reply ansible/roles/minio/tasks/main.yml]: I fixed this.

**Pull request merged**